siad/daemon.go

```diff
@@ -305,6 +306,16 @@ func startDaemonCmd(cmd *cobra.Command, _ []string) {
 		go profile.StartContinuousProfile(globalConfig.Siad.ProfileDir)
 	}
 
+	// Establish a loop that will clear memory every 30 seconds. This will help
+	// keep 'siad' as lightweight as possible. Can be removed once peak usage is
+	// below 1 GiB.
+	go func() {
+		for {
+			debug.FreeOSMemory()
+			time.Sleep(time.Second * 30)
+		}
```

Comment:
for reference: from what I can tell, the runtime will do this every 5 minutes by default